test/e2e/dataplane/fips_status.go

```diff
@@ -0,0 +1,67 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dataplane
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+	subFramework "github.com/submariner-io/submariner/test/e2e/framework"
+)
+
+var _ = Describe("[dataplane] FIPS status reporting", func() {
+	f := subFramework.NewFramework("fips-gateway-status")
+
+	When("a fips mode is configured", func() {
+		It("gateway pod should use the fips mode as part of the libreswan", func() {
+			testFIPSGatewayStatus(f)
+		})
+	})
+})
+
+func testFIPSGatewayStatus(f *subFramework.Framework) {
+	By(fmt.Sprintln("Sanity check - find a cluster with FIPS enabled"))
```

Comment:
We actually expect that FIPS may not be enabled.

```suggestion
	By(fmt.Sprintln("Find a cluster with FIPS enabled"))
```

Reply:
Done